flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSinkTest.java

```diff
@@ -83,7 +103,22 @@ public void throwExceptionIfChannelIsNull() throws Exception {
 	}
 
 	private RMQSink<String> createRMQSink() throws Exception {
-		RMQSink rmqSink = new RMQSink<String>(rmqConnectionConfig, QUEUE_NAME, serializationSchema);
+		RMQSink<String> rmqSink = new RMQSink<String>(rmqConnectionConfig, QUEUE_NAME, serializationSchema);
```

Comment:
nit: Java 7 diamond operator should be preferred, i.e., `new RMQSink<>(...)`

Reply:
Done